nextjs/packages/next/server/config-shared.ts

```diff
@@ -176,6 +189,7 @@ export type NextConfig = { [key: string]: any } & {
     isrMemoryCacheSize?: number
     concurrentFeatures?: boolean
   }
+  codegen?: CodegenConfig
```

Comment:
todo: should be places next to Blitz's stuff starting at `// -- Blitz start`

Reply:
blocker or nitpick?